content/about/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: About
+description: 'About'
+permalink: /about/
+layout: layouts/page
+tags: shareit
+eleventyNavigation:
+  parent: shareit-about
+  key: shareit-about-main
+  order: 1
+  title: About
+sidenav: false
+sticky_sidenav: false
+---
+
+### Agency about
```

Comment:
I would change this to "About the SHARE IT Act"

Reply:
Changed, also changed sidenav to true